src/Document.js

```diff
@@ -187,6 +177,39 @@ export default class Document extends ParentNode {
         return null;
     }
 
+    get documentElement() {
+        return this._documentElement;
+    }
+
+    get head() {
+        let head = this._documentElement.getElementsByTagName('head').shift();
+        if (!head) {
```

Comment:
Well I thought about this, but there were some reason for this. I can't remember that now. As far as I know there is a chance of such situation:
```
let doc = new Document();
assert(doc.innerHTML, '<html></html>') => true;
```

So I covered it with constructing field. I understand that this is not clear, but all my tests works well in such cases.